src/Infrastructure/Multitenancy/FSHTenantInfo.cs

```diff
@@ -39,6 +40,7 @@ public FSHTenantInfo(string id, string name, string? connectionString, string ad
     public string AdminEmail { get; private set; } = default!;
     public bool IsActive { get; private set; }
     public DateTime ValidUpto { get; private set; }
+    public bool IsRoot { get; private set; }
```

Comment:
Why would you need a field in the database for this? When you can imply do `tenantInfo.Id == MultiTenancyConstants.Root.Id`.

Reply:
Because if it stores it in the table the constants are used as the initialization of the system, and then you can change without being wired.
It is still simple to replace with a calculated property as indicated by you:

`
public bool IsRoot => tenantInfo.Id == MultiTenancyConstants.Root.Id;
`

Reply:
This is not how this works... You can't change the TenantId after the fact... every single record in the database has to be updated...